src/codegen/sys/cargo_toml.rs

```diff
@@ -93,6 +93,20 @@ fn fill_empty(root: &mut Table, env: &Env, crate_name: &str) {
         set_string(dep, "package", ext_package);
         set_string(dep, "git", repo_url);
     }
+
+    {
+        let features = upsert_table(root, "features");
+        features.insert(
+            "dox".to_string(),
+            Value::Array(
+                env.config
```

Comment:
I would actually drop this dox_feature_dependencies  configuration as nothing seems to use (gtk-rs-core/gstreamer-rs/gtk4-rs don't which are the largest users of gir) and gir is actually aware of the glib-based dependencies which all have a dox feature so you can generate the list automatically